### PR TITLE
e2e-tests: ensure pod assertions find at least one pod

### PR DESCRIPTION
**Pull request by sthaha**

### CODE REVIEW

**Comment by sthaha** [comment]
/hold

Holding this until https://github.com/openshift/cluster-monitoring-operator/pull/2018 and https://github.com/openshift/cluster-monitoring-operator/pull/2011 are merged.

**Review by marioferh — COMMENTED** [review]
/lgtm

File: test/e2e/framework/assertions.go [hunk 1]
```diff
@@ -329,6 +329,10 @@ func (f *Framework) AssertPodConfiguration(namespace, labelSelector string, asse
 				return fmt.Errorf("%w - failed to get Pods", err)
 			}
 
+			if len(pods.Items) == 0 {
+				return fmt.Errorf("%w - failed to find pods in %s matching %s", err, namespace, labelSelector)
```
[the remark this answers is not in the document]
Comment (sthaha) [reply test/e2e/framework/assertions.go:332]: good catch!!

**Review by simonpasquier — COMMENTED** [review]
/lgtm

**Pull request merged**